speak/voice.py

```diff
@@ -110,7 +111,24 @@ def __cmp__(self, other):
 
 
 def _init_voice_cache():
-    for language, name, dialect in espeak.voices():
+    speech_manager = speech.get_speech_manager()
+    all_voices = speech_manager.get_all_voices()
+    out = []  # Voices that produce sound only
+
+    for lang_code, name in all_voices.items():
+        if(name in ('en-rhotic', 'english_rp', 'english_wmids')):
```

Comment:
Unnecessary parentheses.

Reply:
I have removed them in the next patch.
Thanks